vaadin-date-time-picker-flow/pom.xml

```
@@ -86,12 +86,12 @@
         <dependency>
             <groupId>org.webjars.bowergithub.vaadin</groupId>
             <artifactId>vaadin-button</artifactId>
-            <version>2.2.2</version>
+            <version>2.3.0-alpha1</version>
```

Update to stable version.
```suggestion
            <version>2.3.0</version>
```